src/scripts/main.js

```diff
@@ -1,3 +1,300 @@
+      newInput.setAttribute('name', 'salsary');
+      newInput.setAttribute('type', 'number');
+      newInput.setAttribute('data-qa', 'salary');
+      newInput.setAttribute('id', 'inputSalary');
+      newInput.setAttribute('required', '');
+  }
+  newForm.append(newLabel);
+}
+newForm.append(newButton);
+
+// Event submit form
+const form = document.querySelector('#addForm');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const nameEmployee = document.querySelector('#inputName').value;
+  const position = document.querySelector('#inputPosition').value;
+  const office = document.querySelector('#inputOffice').value;
+  const age = document.querySelector('#inputAge').value;
+  const salary = document.querySelector('#inputSalary').value;
+  const newEmployee = document.createElement('tr');
+
+  if (!validation(nameEmployee, 0)) {
+    return;
+  };
+
+  if (!validation(age, 3)) {
+    return;
+  };
+
+  newEmployee.innerHTML = `
+    <td>${nameEmployee}</td>
+    <td>${position}</td>
+    <td>${office}</td>
+    <td>${age}</td>
+    <td>$${Number(salary).toLocaleString('en-US')}</td>
+  `;
+
+  employeesBody.append(newEmployee);
+
+  pushNotification(10, 10, 'Success!!!',
+    `${nameEmployee} has been added to the list.\n `
+    + 'Keep it up.', 'success');
+
+  form.reset();
+});
+
+// Editing of table cells by double-clicking
+const newCell = document.createElement('input');
+const oldInfo = document.createElement('td');
+
+newCell.classList.add('cell-input');
+
+// Event for double-click
+employeesBody.addEventListener('dblclick', (e) => {
+  newCell.value = e.target.textContent;
+  oldInfo.textContent = e.target.textContent;
+  e.target.replaceChildren(newCell);
+  newCell.focus();
+});
+
+// Event for key Enter
+employeesBody.addEventListener('keyup', (e) => {
+  if (e.code === 'Enter') {
+    save(e);
+  }
+});
+
+// Event for save and data validation
+const save = (e) => {
+  if (validation(newCell.value, e.target.parentElement.cellIndex)) {
+    if (e.target.parentElement.cellIndex === 4) {
+      if (newCell.value[0] !== '$') {
+        newCell.parentElement.textContent
+        = `$${Number(newCell.value).toLocaleString('en-US')}`;
+
+        return;
+      }
+
+      newCell.parentElement.textContent
+        = `$${Number(sumString(newCell.value)).toLocaleString('en-US')}`;
+
+      return;
+    }
+    newCell.parentElement.textContent = newCell.value;
+
+    return;
+  }
+
+  newCell.parentElement.textContent = oldInfo.textContent;
+};
+
+newCell.addEventListener('blur', (save));
+
+// Validation function
+const validation = (info, index) => {
+  if (index === 0 || index === 1 || index === 2) {
+    if (info.length < 4) {
+      pushNotification(150, 10, `${titleSort.children[0]
+        .children[index].textContent} is to short!`,
+      `You enter ${info.length} characters.\n `
+      + 'Please enter more than 4 characters.', 'error');
+
+      return false;
+    }
+  }
+
+  if (index === 3) {
```

Comment:
This way it's hard to understand which field you are checking, don’t use indexes. Better check by input name, and make separate functions for each validation:
```
switch (name) {
  case 'name':
    validateName(info);
    break;
  case 'position':
    validatePosition(info);
    break;
} 
```
Also, would be better to change 'info' to 'value', as info is too broad and may mean other things
And use switch instead of multiple ifs